packages/block-library/src/cover/save.js

```diff
@@ -62,7 +62,6 @@ export default function save( { attributes } ) {
 			? backgroundImageStyles( url )
 			: {} ),
 		backgroundColor: ! overlayColorClass ? customOverlayColor : undefined,
```

Comment:
I think this style attribute needs to be moved to the `span` for consistency with the `edit` function. If I set a custom color, it looks okay in the editor, but the front end renders a solid color on the `div` element instead of the `span` so the opacity doesn't affect it:

| Editor | Front-end |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/136117799-923d1d3c-7cfc-4d85-ae51-3d826eb11e87.png) | ![image](https://user-images.githubusercontent.com/14988353/136117842-9547859b-810e-49af-b385-d5d9c4a9fa6e.png) |

Reply:
Thanks for catching this @andrewserong, it should be fixed now if you want to retest.